test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -263,5 +263,79 @@ public void IncompatibleSourcesWarningShouldBeDisplayedInTheConsoleOnlyWhenRunni
                 this.StdOutputContains(expectedWarningContains);
             }
         }
+
+        [TestMethod]
+        [NetFullTargetFrameworkDataSource]
+        public void ExitCodeShouldReturnOneWhenFailWhenNoTestsFoundParameterSetToTrueAndNoTestAvailableWithFilter(RunnerInfo runnerInfo)
+        {
+            SetTestEnvironment(this.testEnvironment, runnerInfo);
+
+            var assemblyPaths = this.BuildMultipleAssemblyPath("SimpleTestProject2.dll").Trim('\"');
+
+            var arguments = PrepareArguments(assemblyPaths, this.GetTestAdapterPath(), string.Empty, this.FrameworkArgValue, runnerInfo.InIsolationValue);
+            
+            // Setting /TestCaseFilter to random test name, which does not exists
```

Comment:
this just states what is happening. change it to explain why, or remove the comment. The filter is not random btw, it is static. I would rather call it TestNameThatMatchesNoTestInTheAssembly